packages/shared/prisma/schema.prisma

```diff
@@ -422,6 +428,34 @@ enum ScoreSource {
     EVAL
 }
 
+model ScoreConfig {
+    id          String        @id @default(cuid())
+    createdAt   DateTime      @default(now()) @map("created_at")
+    updatedAt   DateTime      @default(now()) @updatedAt @map("updated_at")
+    projectId   String        @map("project_id")
+    project     Project       @relation(fields: [projectId], references: [id], onDelete: Cascade)
+    name        String
+    dataType    ScoreDataType @map("data_type")
+    isArchived  Boolean       @default(false) @map("is_archived")
+    minValue    Float?        @map("min_value")
+    maxValue    Float?        @map("max_value")
+    categories  Json?         @map("categories")
```

Comment:
Future proofing question: we are sure we want to use the `Json` data type for categories rather than a separate relational model, right? It suffices our current use case for sure but trying to future proof. Json is likely simpler/more flexible for maintenance, more performant as we're always fetching the entire object. Con: we're not enforcing data integrity on a schema level. Should not be a concern while only creating scoreConfigs via app, but if we allow creation of configs via API or need scalability for writes (don't see strong use case) might become problematic.  

My view: relational model is an overkill in the mid-term. We can always revisit this conversation when allowing config creation via API, don't see it as a one-way door. 

Thoughts? If no concerns + you agree feel free to resolve comment.

Reply:
agree, I'd keep this as JSON. If we really want to we can still backfill a separate table via a migration if we enforce the structure here.

Reply:
agree